test/lib/Controller.test.ts

```diff
@@ -13,132 +13,147 @@ describe('Controller.ts tests', () => {
 		TestOverrides.initializeTestSetup();
 	});
 
-	it('getAllEngines returns enabled/disabled engines', async() => {
-		const engines: RuleEngine[] = await Controller.getAllEngines();
-		const names: string[] = engines.map(e => e.constructor.name);
-
-		expect(engines.length, names + '').to.equal(10);
-		expect(names).to.contain('JavascriptEslintEngine');
-		expect(names).to.contain('LWCEslintEngine');
-		expect(names).to.contain('TypescriptEslintEngine');
-		expect(names).to.contain('CustomEslintEngine');
-		expect(names).to.contain('PmdEngine');
-		expect(names).to.contain('CustomPmdEngine');
-		expect(names).to.contain('RetireJsEngine');
-		expect(names).to.contain('CpdEngine');
-		expect(names).to.contain('SfgeDfaEngine');
-		expect(names).to.contain('SfgePathlessEngine');
+	describe('#getAllEngines()', () => {
```

Comment:
Reorganized this test file. Nearly every change is moving/renaming a test and changing an expected number of engines from X to X-1.